spread.yaml

```diff
@@ -52,7 +52,7 @@ backends:
       system=$(echo "${SPREAD_SYSTEM}" | tr . -)
       instance_name="spread-${SPREAD_BACKEND}-${instance_num}-${system}"
 
-      multipass launch --cpus 2 --disk 20G --memory 2G --name "${instance_name}" "${multipass_image}"
+      multipass launch --cpus 4 --disk 50G --memory 8G --name "${instance_name}" "${multipass_image}"
```

Comment:
You may want to decrease the `--disk` to 40G, see https://github.com/canonical/charmcraft/pull/1976#discussion_r1868502562